scripts/examples/gn_bouffalolab_example.sh

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env bash
+
+#
+#    Copyright (c) 2020 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+set -e
```

Comment:
please use build_examples.py instead of custom scripts

Reply:
build_examples.py supports below bouffalo lab develop board build with/without pigweed RPC. We also have some special parameters for image building, such as module_type. build_examples.py is not flexibly enough to input such parameters.

bouffalolab-BL702-IoT-DVK-light
bouffalolab-BL702-IoT-DVK-light-rpc
bouffalolab-BL706-IoT-DVK-BL706C-22-light
bouffalolab-BL706-IoT-DVK-BL706C-22-light-rpc
bouffalolab-BL706-NIGHT-LIGHT-light